File: src/main/java/dev/slne/spawn/trader/entity/CustomTrader.java
package dev.slne.spawn.trader.entity;

import org.bukkit.Location;

/**
 * The interface Custom trader.
 */
public interface CustomTrader {
  /**
   * Spawn.
   *
   * @param location the location
   */
  boolean spawn(Location location, String name);

  /**
   * Clear.
   */
  boolean clear(String name);
}

File: src/main/java/dev/slne/spawn/trader/entity/CustomTrader.kt
package dev.slne.spawn.trader.entity

import org.bukkit.Location

/**
 * The interface Custom trader.
 */
interface CustomTrader {
    /**
     * Spawn.
     *
     * @param location the location
     */
    suspend fun spawn(location: Location, name: String): Boolean

    /**
     * Clear.
     */
    fun clear(name: String): Boolean
}
